sdk/ml/azure-ai-ml/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 1.10.1 (2023-09-17)
+
+### Bugs Fixed
+- Feature sets can now be registers after being dumped and reloaded.
+- SDK can now assign materialization identities to offline stores and cross RG online stores.
+
```

Comment:
nit: SDK can now assign materialization identities to cross RG offline stores and online stores.

Reply:
changes